CommunityProjects.md

```diff
@@ -8,30 +8,40 @@ SPDX-FileCopyrightText: 2020 seL4 Project a Series of LF Projects, LLC.
 This page collects seL4-based projects. If you know of a project that
 you'd like to be listed here, add it.
 
+## Application areas
 The seL4 kernel is currently used in projects in the following application areas:
 
 - Automotive
-- Aviation (e.g.,
-        [SMACCM](https://trustworthy.systems/projects/TS/SMACCM/))
```

Comment:
the e.g. there is on purpose, SMACCM is not the only aviation project

Reply:
> the e.g. there is on purpose, SMACCM is not the only aviation project

Ok, I see the point. Would be nice to give at lest two example here. Any ideas?

Reply:
Most of the other ones are not public, but I think we can mention the CASE project at <https://trustworthy.systems/projects/TS/CASE/>